### DISCUSSION TITLE
Get-FalconDetection Closure time

### DISCUSSION THREAD

**Discussion by ninja5244** [CrowdStrike/psfalcon#384] [Q&A]
I want to retrieve closure time detail using psfalcon. using Get-FalconDetection, i could get all other details related to cs detection except closure time. my main objective is to retrieve the open and closure time details of closed detections. Could anyone help me on this.

**Comment by bk-cs** [comment]
I'm not sure that is specifically tracked. There is a field called `seconds_to_resolved` which could be added to `created_timestamp` to determine the specific date/time that a detection was marked as resolved.
